.eslintrc.json

```diff
@@ -1,6 +1,6 @@
 {
-	"extends": "brightspace/polymer-3-config",
-	"env": {
-    "es6": true
+  "extends": "brightspace/lit-config",
+  "rules": {
+    "lit/no-legacy-template-syntax": 0
```

Comment:
Turns out this is the only rule I needed to turn off.  If I target Polymer files with a different eslint config, it's a bit of a random sprinkling:
- `d2l-filter-dropdown` (but not the other filter dropdown components)
- `d2l-search-facets-grouping` and `d2l-search-facets-option` (but not the other search facets component)
- Two demo components
I think I'd like to get this repo fully to Lit, but I want to make sure I get the My Courses filter done for 20.20.10 first.  Is it good to turn off this rule for now?